### ISSUE TITLE
Parse literals without creating a custom token

### ISSUE THREAD

**Issue by vihanb** [Hardmath123/nearley#226]
So right know if I have something like:
```
foo -> "aa"
```

and I run `p.feed(["aa"])` nearley will not be able to parse, this. The docs say I have to do:

```
var matchAa = { literal: "aa" }

...

foo -> %matchAA
```

however is it possible to add the ability for me to just do `"aa"` in the code? With a lot of literals this starts to feel really repetitive an unnecessary.

**Comment by somebody1234** [comment]
Basically stop doing `"symbols": [{"literal":"="}, {"literal":"="}], "postprocess": function joiner(d) {return d.join('');}` for e.g. `"=="`

**Comment by vihanb** [comment]
I see that there is a `@lexer` update that was recently merged that kinda has a solution to this but this isn't on NPM, could the NPM be updated?

**Comment by tjvr** [comment]
That's the plan—waiting for @hardmath122 to have a spare moment to update NPM! :-)

**Comment by tjvr** [comment]
We've now published `2.9.0` with `@lexer` support. :)
